backend/corpora/common/entities/dataset_asset.py

```diff
@@ -10,8 +10,6 @@
 from ..corpora_orm import DbDatasetArtifact, DatasetArtifactFileType
 from ..utils.s3_buckets import buckets
 
-logger = logging.getLogger(__name__)
```

Comment:
Using the root logger so the json logging changes take effect.

Reply:
Does this mean you'll no longer see what file the logging message is coming from?

Reply:
It does. Before this would have said in the logs {"name": "dataset_asset"}, now it will be {"name":"root"}. We can add pathname to the log record. That will include the full path to the file that produced the log.